cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt

```diff
@@ -592,8 +601,8 @@ script "js_azure_sql_database_metrics", type: "javascript" do
       "api-version": "2018-01-01",
       "timespan": timespan,
       "metricnames": "cpu_percent,connection_successful",
-      "aggregation": "Average,count",
-      "interval":  "P1D"
+      "aggregation": "Average,Minimum,Maximum,count",
+      "interval": "P1D"
```

Comment:
This should be a parameter imo... The a smaller interval would return more data points, and improve the accuracy of the percentile results.  Default in the Databricks policy template has 15min interval by default

relevant snippets:

The param block:
https://github.com/flexera-public/policy_templates/blob/8adb71e4b2a0b08de688987c159cc9a94f663a80/cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt#L100-L107

The ds to map param value to api payload value:
https://github.com/flexera-public/policy_templates/blob/master/cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt#L384-L405

The request:
https://github.com/flexera-public/policy_templates/blob/master/cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt#L587

Reply:
An example of the results we can get with 1 day interval (only 30 days of data returns * 3 statistics == 90 total points
<img width="1159" alt="image" src="https://github.com/flexera-public/policy_templates/assets/1490015/e6129049-a4bd-4c44-957a-cca5b63db9c3">

If 9 days of the default 30 lookback period had max:100% cpu, then p90, p90, p99 would == 100.
> .90 * 90 == 81
> For a list of 90 values (min+avg+max measurement * 30 days), the value at index 81 would get selected for p90.. and with 100 being max, so would p95 and p99.
The list of measurements is sorted by maximums so only 9 days out of the 30 would need to have hit max:100% cpu during the day to get this result

With more measurements, we get a better estimate of the underlying distribution. When the sample size is small (e.g., less than 100), percentiles can be sensitive to outliers or extreme values. Larger samples mitigate this issue.

Reply:
Would this be within the scope of this ticket, or is this a broader ask that would apply to other policies as well?

Just trying to identify scope and whether this change needs to be part of *this* work or not.